timeseries/setup.py

```diff
@@ -47,14 +45,14 @@
     )
 
 extras_require = {
-    "tests": ["pytest", "flake8~=4.0", "flaky~=3.7", "pytest-timeout~=2.1"],
+    "tests": ["pytest", "flake8~=4.0", "flaky~=3.7", "pytest-timeout~=2.1", "isort>=5.10"],
```

Comment:
do we need to add this to requirements? Isn't this only necessary at development time?

Reply:
I don't think we need it here but we need to mention it somewhere so that people can automatically lint the code they develop. I followed the logic in [`text/setup.py`](https://github.com/awslabs/autogluon/blob/master/text/setup.py) and [`multimodal/setup.py`](https://github.com/awslabs/autogluon/blob/master/multimodal/setup.py) where `black` is mentioned as a requirement in the `tests` scope. Is there some other place where we could move this dependency?

Reply:
I see. But our tests don't depend on `black` or `isort` and here we don't have "black" anyway? I would either move these to `CONTRIBUTION.md` or put black also here?

Reply:
Could we maybe add black to test requirements also to trigger CI again?